In [33]:
from scipy import stats
import numpy as np

#### 2) Проведите тест гипотезы. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв. мм.

In [1]:
(17.5- 17)/(2/10)

2.5

Вывод: выбираем альтернативную гипотезу

#### 3)Проведите тест гипотезы. Продавец утверждает, что средний вес пачки печенья составляет 200 г.Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%? (Провести двусторонний тест.)

In [40]:
x = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
print(x.mean())
print(x.std(ddof=1))
t = (x.mean()-200)/(x.std(ddof=1)/np.sqrt(10))
print(t)
t_table = stats.t.ppf(0.005, 9)
t_table

198.5
4.453463071962462
-1.0651074037450896


-3.24983554401537

Наблюдаемое значение не выходит за рамки -3.25 < t <  3.25. Утверждения продавца верны

#### 4)Есть ли статистически значимые различия в росте дочерей? Рост матерей 172, 177, 158, 170, 178,175, 164, 160, 169 Рост взрослых дочерей: 173, 175, 162, 174, 175, 168, 155, 170, 160

In [48]:
x = np.array([172, 177, 158, 170, 178,175, 164, 160, 169])
y = np.array([173, 175, 162, 174, 175, 168, 155, 170, 160])
stats.ttest_ind(x,y)


Ttest_indResult(statistic=0.35581195161541973, pvalue=0.7266292679160495)

Pvalue > $\alpha$, Статистических различий нет